engine/render/src/render/font.cpp

```diff
@@ -139,15 +124,49 @@ namespace dmRender
         }
     }
 
-    void SetFontMap(HFontMap font_map, dmRender::HRenderContext render_context, dmGraphics::HContext graphics_context, FontMapParams& params)
+    static void RecreateTexture(HFontMap font_map, dmGraphics::HContext graphics_context, uint32_t width, uint32_t height)
     {
+        // create new texture to be used as a cache
+        dmGraphics::TextureCreationParams tex_create_params;
+        tex_create_params.m_Width = width;
+        tex_create_params.m_Height = height;
+        tex_create_params.m_OriginalWidth = width;
+        tex_create_params.m_OriginalHeight = height;
+
+        dmGraphics::TextureParams tex_params;
+        tex_params.m_Format = font_map->m_CacheFormat;
+        tex_params.m_Data = 0;
+        tex_params.m_DataSize = 0;
+        tex_params.m_Width = width;
+        tex_params.m_Height = height;
+        tex_params.m_MinFilter = dmGraphics::TEXTURE_FILTER_LINEAR;
+        tex_params.m_MagFilter = dmGraphics::TEXTURE_FILTER_LINEAR;
+
+        if (font_map->m_Texture)
+        {
+            dmGraphics::DeleteTexture(font_map->m_Texture);
+        }
+        font_map->m_Texture = dmGraphics::NewTexture(graphics_context, tex_create_params);
```

Comment:
Perhaps there's a more efficient way of resizing a texture.
Also, a bit annoying I need the graphics context here :/

Reply:
We use the context for both storing the actual texture object, as well as checking certain context features such as texture arrays for paged atlases. For the graphics adapters we haven't really picked a lane for how to deal with the context, we kind of mix passing in a context and using the global context per adapter so the jury is out on what we _should_ do I think